### ISSUE TITLE
Feature: User credentials for service widget

### ISSUE THREAD

**Issue by Khaos66** [AnyStatus/Support#87]
If the current user doesn't have the privilege to check if a windows service is running you need admin credentials

**Comment by devsachi** [comment]
I agree, it would be nice if we can connect to any windows service machine using runAs.

**Comment by AlonAm** [comment]
Our plan is to add a new widget called Remote Windows Service that will act the same as the local version. I'm looking for solutions to achieve that with minimal dependencies. You're welcome to suggest possible solutions.

**Comment by AlonAm** [comment]
First, the existing implementation uses an API that doesn't allow specifying all required credentials (I think). Thus, we need to find a different way to monitor and control remote and/or local services. Maybe using SC.exe or WMI instead.

Secondly, I suggest porting the widget to AnyStatus v3 use the new Endpoints feature to store the credentials once for each remote/local machine.
